janus_core/calculations/geom_opt.py

```diff
@@ -160,6 +161,13 @@ def __init__(  # pylint: disable=too-many-arguments
         self.write_kwargs = write_kwargs
         self.traj_kwargs = traj_kwargs
 
+        if not isinstance(struct, Atoms):
```

Comment:
Actually, I see now that that's actually forbidden. So surely the first check is enough.

Reply:
Which do you mean by the first check?

The aim here was to say that struct has to be a single `Atoms` object, but if it's a sequence of `Atoms` objects, throw a slightly different error, since it's valid for other calculations, whereas if it was a string for example, that's just completely wrong.

Reply:
Fair enough.